test-examples/jdi-bdd-no-po-tests/src/test/java/cucmberTests/TestRunner.java

```diff
@@ -1,27 +1,28 @@
 package cucmberTests;
 
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+import static com.epam.jdi.light.elements.init.UIFactory.form;
+import static cucmberTests.test.data.TestData.ROMAN;
+
 import com.epam.jdi.light.elements.composite.WebPage;
 import cucmberTests.test.data.User;
 import cucumber.api.CucumberOptions;
 import cucumber.api.junit.Cucumber;
-import cucumber.api.testng.AbstractTestNGCucumberTests;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.runner.RunWith;
 
-import static com.epam.jdi.light.driver.WebDriverUtils.*;
-import static com.epam.jdi.light.elements.init.UIFactory.*;
-import static cucmberTests.test.data.TestData.*;
-
 @RunWith(Cucumber.class)
 @CucumberOptions(
     features = "classpath:features"
     , glue = {"com.epam.jdi.bdd", "cucmberTests"}
     , tags = {"@form"}
 )
-public class TestRunner extends AbstractTestNGCucumberTests {
+public class TestRunner {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestRunner' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5524853)